GitCommands/Git/GitModule.cs

```diff
@@ -3733,6 +3733,46 @@ public static string UnescapeOctalCodePoints([CanBeNull] string s)
                 });
         }
 
+        /// <summary>
+        /// Escapes a UTF8 string <paramref name="s"/> into octal code points.
+        /// </summary>
+        /// <remarks>
+        /// If <paramref name="s"/> is <c>null</c> then an empty string is returned.
+        /// </remarks>
+        /// <example>
+        /// <code>EscapeOctalCodePoints("두다") == @"\353\221\220\353\213\244"</code>
+        /// </example>
+        /// <param name="s">The string to escape.</param>
+        /// <returns>The escaped string, or <c>""</c> if <paramref name="s"/> is <c>null</c>.</returns>
+        public static string EscapeOctalCodePoints([CanBeNull] string s)
+        {
+            if (s == null)
+            {
+                return null;
+            }
+
+            var resultBuilder = new StringBuilder();
```

Comment:
nitpick perf: could you initialize the StringBuilder with the length of the original string?

```suggestion
            var resultBuilder = new StringBuilder(s.Length);
```